File: app/src/main/java/com/example/test/MenuFragment.java
package com.example.test;

import android.content.Intent;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.LinearLayout;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import android.widget.TextView;

public class MenuFragment extends Fragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_menu, container, false);

        // Tampilkan username di greeting
        TextView tvHello = view.findViewById(R.id.r3tii9i8wowm);
        String username = getArguments() != null ? getArguments().getString("USER_NAME") : "User";
        tvHello.setText("Hello, " + username);

        // Tambahkan import TextView di atas
        LinearLayout btnTambahSateAyam = view.findViewById(R.id.btnTambahSateAyam);
        btnTambahSateAyam.setOnClickListener(v -> startActivity(new Intent(getActivity(), Checkout.class)));

        LinearLayout btnTambahNasiGoreng = view.findViewById(R.id.btnTambahNasiGoreng);
        btnTambahNasiGoreng.setOnClickListener(v -> startActivity(new Intent(getActivity(), Checkout.class)));

        LinearLayout btnTambahSopBuntut = view.findViewById(R.id.btnTambahSopBuntut);
        btnTambahSopBuntut.setOnClickListener(v -> startActivity(new Intent(getActivity(), Checkout.class)));

        LinearLayout btnTambahNasiGorengDewa = view.findViewById(R.id.btnTambahNasiGorengDewa);
        btnTambahNasiGorengDewa.setOnClickListener(v -> startActivity(new Intent(getActivity(), Checkout.class)));

        return view;
    }
}

File: app/src/main/java/com/example/test/MenuFragment.kt
package com.example.test

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.LinearLayout
import android.widget.TextView
import androidx.fragment.app.Fragment

class MenuFragment : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val view = inflater.inflate(R.layout.fragment_menu, container, false)

        // Tampilkan username di greeting
        val tvHello = view.findViewById<TextView>(R.id.r3tii9i8wowm)
        val username =
            if (getArguments() != null) getArguments()!!.getString("USER_NAME") else "User"
        tvHello.setText("Hello, " + username)

        // Tambahkan import TextView di atas
        val btnTambahSateAyam = view.findViewById<LinearLayout>(R.id.btnTambahSateAyam)
        btnTambahSateAyam.setOnClickListener(View.OnClickListener { v: View? ->
            startActivity(
                Intent(
                    getActivity(),
                    Checkout::class.java
                )
            )
        })

        val btnTambahNasiGoreng = view.findViewById<LinearLayout>(R.id.btnTambahNasiGoreng)
        btnTambahNasiGoreng.setOnClickListener(View.OnClickListener { v: View? ->
            startActivity(
                Intent(getActivity(), Checkout::class.java)
            )
        })

        val btnTambahSopBuntut = view.findViewById<LinearLayout>(R.id.btnTambahSopBuntut)
        btnTambahSopBuntut.setOnClickListener(View.OnClickListener { v: View? ->
            startActivity(
                Intent(getActivity(), Checkout::class.java)
            )
        })

        val btnTambahNasiGorengDewa = view.findViewById<LinearLayout>(R.id.btnTambahNasiGorengDewa)
        btnTambahNasiGorengDewa.setOnClickListener(View.OnClickListener { v: View? ->
            startActivity(
                Intent(getActivity(), Checkout::class.java)
            )
        })

        return view
    }
}